rpc-server.py

```diff
@@ -129,8 +138,8 @@ def run_rpc_server(port, no_token):
     print(app_port, file=sys.stdout, flush=True)
 
     # Redirect stdout and stderror to log files
-    sys.stdout = open('rpc-server.out', 'w')
-    sys.stderr = open('rpc-server.err', 'w')
+    # sys.stdout = open('rpc-server.out', 'w')
+    # sys.stderr = open('rpc-server.err', 'w')
```

Comment:
@ivanalejandro0, It's not a problem to comment it for development, but I think that it should be better to have an argument for redirecting stdout/stderr to separate files. In the future, the **gui** is going to load the rpc server and it'd be nice to have all the logs. Do you have something else in mind?
Of course, It's neither a priority nor a show stopper.